frontend/src/store/FamLoginUserState.ts

```diff
@@ -106,6 +110,21 @@ const getApplicationsUserAdministers = () => {
     return applicationList;
 };
 
+const isApplicationAdmin = () =>
+    getApplicationsUserAdministers().some(
+        (application) => application.id === selectedApplicationId.value
+    );
+
```

Comment:
so getApplicationsUserAdministers will return an application list the user has access to, the user can be either delegated admin or app admin of those applications. And your filter here can only confirm user has access to the selected app, but can't confirm he's the app admin, it's also possible the user is the delegated admin of the selected app. 

So could do something like
```
const appsUserIsAdmin = accesses?.filter(access => access.auth_key == 'APP_ADMIN')
appsUserIsAdmin[0].grants.filter(grant => grant.application.id == FAM_APPLICATION_ID)
```
check if the 2nd filter result has length > 0

Reply:
Hi Ian, just knowing user has APP_ADMIN role is not enough, when user selects an application from the dropdown, we need to know if user is the admin of this specific selected application, and then we can decide if display delegated admin table or not.

Reply:
maybe rename it to be `isAdminOfSelectedApplication`

Reply:
Yes, thanks Catherine!
 If the function's purpose was not initially what it appears to me, then I suggest @J0taFerreira when creating a function, try (sometime is hard) pay a bit attention on how it sounds to you to name it and also please provided function level documentation so it is clear (hopefully) to others for the purpose of the function.